src/battle_anim_effects_1.c

```diff
@@ -4734,7 +4734,7 @@ static void AnimFalseSwipeSlice(struct Sprite *sprite)
 
 static void AnimFalseSwipePositionedSlice(struct Sprite *sprite)
 {
-    sprite->x = sprite->x = GetBattlerSpriteCoord(gBattleAnimTarget, BATTLER_COORD_X_2) + 0xFFD0 + gBattleAnimArgs[0];
+    sprite->x = GetBattlerSpriteCoord(gBattleAnimTarget, BATTLER_COORD_X_2) + 0xFFD0 + gBattleAnimArgs[0];
```

Comment:
While you're here, that should be `-48` not `0xFFD0`

Reply:
Done, though changing the one in `AnimFalseSwipeSlice` doesn't match.

Reply:
In `AnimFalseSwipeSlice` it needs to explicitly be `-48`, so `GetBattlerSpriteCoord(gBattleAnimTarget, BATTLER_COORD_X_2) + -48` or `-48 + GetBattlerSpriteCoord(gBattleAnimTarget, BATTLER_COORD_X_2)` match. Don't feel like you need to go back and do that though, given it's unrelated to the PR.